site/src/components/table.md

```diff
@@ -122,4 +133,202 @@ _`table-stacked-mobile`_ - stacks table cells on mobile screen sizes.
 		</tr>
 	</tbody>
 </table>
-```
+
+### Complex
+
+_`table-complex`_ - complex table style is a completely bordered cell layout with a heavy stylized header row.
```

Comment:
I've got a little recommendation here to align how we talk about `table_complex` a bit more similarly to how we talk about the other classes:

```suggestion
_`table-complex`_ gives the table a completely bordered cell layout with a heavily stylized header row.
```

Reply:
Updated